.travis.yml

```diff
@@ -13,15 +32,14 @@ julia:
   - 1.0
   - nightly
 env:
-  - JULIA_PROJECT="@."
+  - JULIA_TZ_VERSION="2016j"  # Ideally only set for the "test" stage
 notifications:
   email: false
-env:
-  - JULIA_TZ_VERSION="2016j"
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
   # Avoid setting JULIA_TZ_VERSION if the deps/build.jl file has been modified
   - git fetch origin +:refs/remotes/origin/HEAD; if ! git diff --quiet origin/HEAD HEAD -- deps/build.jl; then unset JULIA_TZ_VERSION; fi
+  - echo $JULIA_TZ_VERSION  # Display version here as Julia 0.7+ doesn't show the build logs by default
```

Comment:
Added as it's hard to verify that this variable is correctly being unset now that the `Pkg.build` output is suppressed. @ararslan mentioned with Julia 1.1 you can use `Pkg.build(verbose=true)`

Reply:
That actually happens by default using the default Travis script with 1.1 or later. (Or rather it will once Fredrik's Travis PR is merged and deployed.)